src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,62 @@
+/* eslint-disable max-len */
 import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+import { Movie } from '../../types/Movie';
+
+interface CardProps {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<CardProps> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={imgUrl}
+            alt="Film logo"
```

Comment:
Lets do it this way for consistency :)
```suggestion
            alt="Movie logo"
```